internal/cosmos-scripts/configs/webpack.development.config.js

```diff
@@ -17,7 +17,8 @@ module.exports = {
   },
   devtool: 'cheap-module-source-map',
   devServer: {
-    contentBase: path.resolve(process.cwd(), 'public')
+    contentBase: path.resolve(process.cwd(), 'public'),
+    disableHostCheck: true
```

Comment:
Temporary measure to disable host checks, we should upgrade webpack-dev-server (Reference: https://github.com/webpack/webpack-dev-server/issues/1604)

Reply:
Opened an issue for that: #1440